core/contextmenu.js

```diff
@@ -10,93 +10,118 @@
  */
 'use strict';
 
-/**
- * @name Blockly.ContextMenu
- * @namespace
- */
-goog.provide('Blockly.ContextMenu');
+goog.module('Blockly.ContextMenu');
+goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.browserEvents');
-goog.require('Blockly.Events');
+// TODO(#5073): Add Blockly require after fixing circular dependency.
+// goog.require('Blockly');
+const Block = goog.requireType('Blockly.Block');
+const WorkspaceSvg = goog.requireType('Blockly.WorkspaceSvg');
+const Coordinate = goog.require('Blockly.utils.Coordinate');
+const Events = goog.require('Blockly.Events');
+const Menu = goog.require('Blockly.Menu');
+const MenuItem = goog.require('Blockly.MenuItem');
+const Msg = goog.require('Blockly.Msg');
+const Rect = goog.require('Blockly.utils.Rect');
+const WidgetDiv = goog.require('Blockly.WidgetDiv');
+const Xml = goog.require('Blockly.Xml');
+const aria = goog.require('Blockly.utils.aria');
+const browserEvents = goog.require('Blockly.browserEvents');
+const dom = goog.require('Blockly.utils.dom');
+const internalConstants = goog.require('Blockly.internalConstants');
+const userAgent = goog.require('Blockly.utils.userAgent');
+const utils = goog.require('Blockly.utils');
 /** @suppress {extraRequire} */
 goog.require('Blockly.Events.BlockCreate');
-goog.require('Blockly.internalConstants');
-goog.require('Blockly.Menu');
-goog.require('Blockly.MenuItem');
-goog.require('Blockly.Msg');
-goog.require('Blockly.utils');
-goog.require('Blockly.utils.aria');
-goog.require('Blockly.utils.Coordinate');
-goog.require('Blockly.utils.dom');
-goog.require('Blockly.utils.Rect');
-goog.require('Blockly.utils.userAgent');
-goog.require('Blockly.WidgetDiv');
-goog.require('Blockly.Xml');
-
-goog.requireType('Blockly.Block');
-goog.requireType('Blockly.WorkspaceSvg');
 
 
 /**
  * Which block is the context menu attached to?
- * @type {Blockly.Block}
+ * @type {Block}
```

Comment:
I think all three of these should be {?Block} since the '?' is no longer optional under the new style guide.